assets/sass/okta-sign-in.scss

```diff
@@ -73,13 +72,16 @@
     }
   }
 
+
+  @import 'modules/accessibility';
```

Comment:
Was this moved down intentionally?

Reply:
Yes, see changes section in the PR description :)